cmd/ttn-lw-cli/commands/organizations_access.go

```diff
@@ -85,7 +85,11 @@ var (
 			if orgID == nil {
 				return errNoOrganizationID
 			}
-			collaborator := getUserID(cmd.Flags(), nil).OrganizationOrUserIdentifiers()
+			usrID, err := getUserID(cmd.Flags(), nil)
+			if err != nil {
+				return err
+			}
+			collaborator := usrID.OrganizationOrUserIdentifiers()
```

Comment:
Can remove this check and just use `usrID.OrganizationOrUserIdentifiers()` directly. Also below.